ml-agents/mlagents/trainers/demo_loader.py

```diff
@@ -28,10 +28,9 @@ def make_demo_buffer(
     # Create and populate buffer using experiences
     demo_raw_buffer = AgentBuffer()
     demo_processed_buffer = AgentBuffer()
-    for idx, experience in enumerate(pair_infos):
+    for idx, current_pair_info in enumerate(pair_infos):
```

Comment:
@kevtan Heads up on this, might cause merge conflicts with what you're working on.